ledger-api/grpc-definitions/com/daml/ledger/api/v1/transfer.proto

```diff
@@ -33,17 +33,18 @@ message Transfer {
   // Optional
   string workflow_id = 3;
```

Comment:
do we have a usecase for workflow ids for transfers? I suppose you can argue that they are not really used for command submissions either so maybe keeping it consistent for now makes sense.